src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1092,12 +1084,6 @@ private void SaveServerCredentialsToCache(Uri uri, uint authScheme, ICredentials
                 return;
             }
 
-            var cancellationTokenRegistration = state.CancellationToken.Register(s =>
```

Comment:
I presume this is by design: cancellation won't be possible after the request is in flight?